DEVELOPMENT.md

```diff
@@ -1 +1 @@
-This file was moved to `CONTRIBUTING.md`.
+This file was moved to <https://github.com/kubereboot/kured/blob/main/CONTRIBUTING.md>.
```

Comment:
Rather than link this to a branch (who knows, we could change `main` at some point) we should use a relative link. E.g.:

```markdown
This file was moved to [CONTRIBUTING.md](CONTRIBUTING.md).
```